xapian-bindings/ruby/ruby.i

```diff
@@ -111,6 +111,11 @@
 %rename("wdf") get_wdf;
 %rename("weight") get_weight;
 
+// Name of constants should start wth upper_case
```

Comment:
Comment typo: "wth"

Also, "upper_case" is a bit weird here - "upper case" with a space would be better.

Reply:
Fixed